### DISCUSSION TITLE
Want to read data from Powermeter from fhem device

### DISCUSSION THREAD

**Discussion by ADecki2902** [hoylabs/OpenDTU-OnBattery#1074] [General]
Hello, I have the powermeter added to my fhem installation and it is working well. Now I want to add Opendtu oB to read the realtime value power from fhem. Does anyone know, how to do this? It is not a json nor a mqtt device. (Volkszähler)

**Comment by spcqike** [comment]
does this help?

https://github.com/helgeerbe/OpenDTU-OnBattery/issues/359#issuecomment-1667391372

**Reply by ADecki2902** [reply]
No,
With curl can read the powermeter and I saw the neeeded value.

> curl http://192.168.178.XXX:8083/fhem?detail=Stromzaehler
I get this:
```
<tr class="even"><td><div class="dname" data-name="Stromzaehler">power</div></td>
<td><div class='dval' class='dval' informId='Stromzaehler-power'>170.4</div></td>
<td><div informId='Stromzaehler-power-ts'>2024-06-30 20:36:01</div></td>
</tr>
```
I need the Stromzaehler-power value, but I´m not confirm with html.
This is only html but no JSON. Is it possible to use?

**Reply by spcqike** [reply]
No. You can’t use html 

as you mentioned Volkszähler, I thought you can use this as this gives you a json as shown in my link 

however you can use FHEM. Setup a MQTT broker and connect FHEM to it. Publish your powermeter readings frequently to MQTT. 
connect your openDTU to the same server and listen to said published readings
